File: src/codes/Users_Manual/sampling_config/modulation_0.py
~from datetime import timedelta
~from pyautd3 import Hz
~from pyautd3.modulation import Sine
m = Sine(150 * Hz).with_sampling_config(4000 * Hz)
# or
m = Sine(150 * Hz).with_sampling_config(timedelta(microseconds=250))

File: src/codes/Users_Manual/sampling_config/modulation_0.cpp
//~#include<chrono>
//~#include<autd3.hpp>
//~#include<autd3/link/nop.hpp>
//~int main() {
//~{
const auto m = autd3::modulation::Sine(150 * autd3::Hz)
                   .with_sampling_config(4000 * autd3::Hz);
//~}
// or
//~{
const auto m = autd3::modulation::Sine(150 * autd3::Hz)
                   .with_sampling_config(std::chrono::microseconds(250));
//~}
//~return 0; }